starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+  font-weight:lighter;
+}
+h1 {
+  background:url(/images/apple-pie.jpg);
```

Comment:
to get the image to show you need to this : 
`  background:url("../images/apple-pie.jpg");` 

this is because you need to go up one level in your folder structure